db/go/20201031165106_add_tensorboard_env_var_to_jupyterlab_template.go

```diff
@@ -0,0 +1,26 @@
+package migration
+
+import (
+	"database/sql"
+	"github.com/pressly/goose"
+)
+
+func initialize20201031165106() {
+	if _, ok := initializedMigrations[20201031165106]; !ok {
+		goose.AddMigration(Up20201031165106, Down20201031165106)
+		initializedMigrations[20201031165106] = true
+	}
+}
+
+// Up20201031165106 updates the jupyterlab workspace to include container lifecycle hooks.
+// These hooks will attempt to persist conda, pip, and jupyterlab extensions between pause and shut-down.
+func Up20201031165106(tx *sql.Tx) error {
+	// This code is executed when the migration is applied.
+	return updateWorkspaceTemplateManifest("20201031165106_jupyterlab.yaml", jupyterLabTemplateName)
```

Comment:
@aleksandrmelnikov @Vafilor I think for the YAML name, we should have the timestamp first so things are ordered in the IDE accordingly. Thoughts?

Reply:
@rushtehrani Good point, yeah, let's do that.

Reply:
Agreed